internal/server/data/data.go

```diff
@@ -169,12 +169,22 @@ type UniqueConstraintError struct {
 }
 
 func (e UniqueConstraintError) Error() string {
-	if e.Table == "" {
+	table := e.Table
+	switch table {
+	case "":
 		return "value already exists"
-	} else if e.Column == "" {
-		return fmt.Sprintf("value already exists for %v", e.Table)
+	case "identities":
+		table = "user"
+	case "access_keys":
+		table = "access key"
+	default:
+		table = strings.TrimSuffix(table, "s")
 	}
-	return fmt.Sprintf("value for %v already exists for %v", e.Column, e.Table)
+
+	if e.Column == "" {
+		return fmt.Sprintf("a %v with that value already exists", table)
```

Comment:
Maybe `UniqueConstraintError` can have a default Column value of `value` so as to not need this branch?

Reply:
Ya, that could work. Right now we create the type in two places, so we'd have to set that default in both places.

I think my preference would be to keep this branch so that we don't have to remember to set a default in every place we create the error, and so that we don't need a constructor to set it.